com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -1008,23 +1037,31 @@ internal void UpdateNetworkTime(ulong clientId, float netTime, float receiveTime
 
             if (NetworkLog.CurrentLogLevel <= LogLevel.Developer)
             {
-                NetworkLog.LogInfo($"Received network time {netTime}, RTT to server is {rtt}, {(warp ? "setting" : "smearing")} offset to {m_NetworkTimeOffset} (delta {m_NetworkTimeOffset - m_CurrentNetworkTimeOffset})");
+                NetworkLog.LogInfo(
+                    $"Received network time {netTime}, RTT to server is {rtt}, {(warp ? "setting" : "smearing")} offset to {m_NetworkTimeOffset} (delta {m_NetworkTimeOffset - m_CurrentNetworkTimeOffset})");
             }
         }
 
         private void SendConnectionRequest()
         {
-            using (var buffer = PooledNetworkBuffer.Get())
-            using (var writer = PooledNetworkWriter.Get(buffer))
-            {
-                writer.WriteUInt64Packed(NetworkConfig.GetConfig());
-
-                if (NetworkConfig.ConnectionApproval)
+            var clientIds = new[] {ServerClientId};
+            var context = MessageQueueContainer.EnterInternalCommandContext(
+                MessageQueueContainer.MessageType.ConnectionRequest,
+                NetworkChannel.Internal,
+                clientIds,
+                NetworkUpdateStage.EarlyUpdate
+            );
```

Comment:
💯 
Super-cool addition to sending internal messages... get a context...write things...done!